lib/oref0-setup/device.json

```diff
@@ -17,7 +17,7 @@
     "name": "ns-glucose",
     "extra": {
       "fields": "",
-      "cmd": "bash -c \"curl --compressed -m 30 -s $NIGHTSCOUT_HOST/api/v1/entries/sgv.json?count=1000 | json -e \\\"this.glucose = this.sgv\\\"\"",
+      "cmd": "bash -c \"oref0-get-ns-entries cgm/ns-glucose.json $NIGHTSCOUT_HOST $API_SECRET\"",
```

Comment:
Can we just delete this “device” instead of updating it?

Reply:
Sure if it's not used anywhere